subiquity/common/filesystem/sizes.py

```diff
@@ -75,3 +82,34 @@ def get_efi_size(available_space):
 def get_bootfs_size(available_space):
     all_factors = (uefi_scale, bootfs_scale, rootfs_scale)
     return scale_partitions(all_factors, available_space)[1]
+
+
+def calculate_guided_resize(part_min: int, part_size: int, install_min: int,
+                            part_align: int = 1 << 20) -> GuidedResizeValues:
```

Comment:
Can we spell 1<<20 as MiB and 1<<30 as GiB do you think?

Reply:
Done for the code, left tests alone.